apiary.apib

```diff
@@ -2617,6 +2636,108 @@ Get a list of lost reasons for deals.
                 + id: `6e7fe84d-d4b3-4723-abae-9bc082d8f65a` (string)
                 + name: `Deal declined` (string)
 
+## Deal Pipelines [/dealPipelines]
+
+Each deal belongs to a specific pipeline, in which it goes through the different deal phases.
+
+*Required scopes: `deals`*
+
+### dealPipelines.list [POST /dealPipelines.list]
+
+Get a list of all deal pipelines.
+
++ Request (application/json)
+    + Attributes (object)
+        + filter (object, optional)
+            + `ids`: `46156648-87c6-478d-8aa7-1dc3a00dacab`, `46156648-87c6-478d-8aa7-1dc3a00daca4` (array[string])
+            + status (array, optional)
+                + (enum)
+                    + Members
+                        + open
+                        + pending_deletion
```

Comment:
I've made the other occurences consistent with this one, because of an apiary rendering issue preventing this alternative notation to be displayed correctly.